libs/oidc/provider/src/lib/adapters/oidc.adapter.ts

```diff
@@ -163,26 +146,36 @@ export class OidcAdapter {
   }
 
   public async destroy(id: KindOfId) {
-    const repo = this.connection.getRepository<IRequiredEntityAttrs>(this.name);
+    const repo = this.connection.getRepository(this.repository);
 
     await repo
       .delete({
-        grantId: `${id}`
+        id: `${id}` // Old v6 Adapter used grantId: `${id}` instead of id
       })
       .catch((typeOrmErr) => {
         throw typeOrmErr;
       });
   }
 
   public async revokeByGrantId(grantId: KindOfId) {
-    const repo = this.connection.getRepository<IRequiredEntityAttrs>(this.name);
+    const repo = this.connection.getRepository<IRequiredEntityAttrs>(this.repository);
 
     await repo.delete({
       grantId: `${grantId}`
     });
   }
 }
 
+interface IOidcAdapter {
+  upsert(id, payload, expiresIn);
+  find(id);
+  findByUserCode(userCode);
+  findByUid(uid);
+  destroy(id);
+  revokeByGrantId(grantId);
+  consume(id);
+}
+
```

Comment:
Nothing else implements this interface other than the single `OidcAdapter`. Are there cases where this will be implemented elsewhere?

Reply:
I don't believe so. Just remove it?